modules/eap-xp-20-env/2.0.0/module.yaml

```diff
@@ -0,0 +1,44 @@
+schema_version: 1
+
+name: "eap-xp-20-env"
+description: "JBoss Enterprise Application Platform expansion pack 2.0 env and labels"
+version: "2.0.0"
+
+labels:
+    - name: "org.jboss.product"
+      value: "eap-xp"
+    - name: "org.jboss.product.version"
+      value: "2.0.0"
+    - name: "org.jboss.product.eap.version"
+      value: "7.3.4"
+    - name: "com.redhat.deployments-dir"
+      value: "/opt/eap/standalone/deployments"
+    - name: "com.redhat.dev-mode"
+      value: "DEBUG:true"
+      description: "Environment variable used to enable development mode (debugging). A value of true will enable development mode."
+    - name: "com.redhat.dev-mode.port"
+      value: "DEBUG_PORT:8787"
+      description: "Environment variable used to specify the debug port. If not set, the default EAP debug port will be used (8787). Only applicable when development mode is enabled."
+envs:
+    - name: "WILDFLY_VERSION"
+      value: "7.3.4.GA-redhat-00003"
```

Comment:
Shouldn't the version for XP2 should be `2.0.0.GA-redhat-00002` according to https://code.engineering.redhat.com/gerrit/gitweb?p=jbossas/jboss-eap7.git;a=blob;f=pom.xml;h=6fa374f58f64513a0221b077e319dac3903337ee;hb=be516742047a4ed0883fe08f7d46d07c296f3b9b#l38 ?

Reply:
@dbkreling , yes 2.0.0.

Reply:
Thanks, updated.